golang/server.go

```diff
@@ -82,7 +82,7 @@ func exampleDefinitions() exampleList {
 	exList := exampleList{}
 
 	// The source constants are generated by `go generate` in the sources.go file.
-	exList["mysql"] = &exampleDef{Name: "MySQL", callback: examples.MySQL, Source: mysql}
+	exList["mysql"] = &exampleDef{Name: "UsageExampleMySQL", callback: examples.UsageExampleMySQL, Source: mysql}
```

Comment:
I don't think you need the `Name` to be "UsageExampleMySQL" right? Just "MySQL" like the other language examples? https://pr-27-nqbfuoq-rjify4yjcwxaa.eu-3.platformsh.site/golang/